tools/DevDiagnostics/DevHome.DevDiagnostics/Models/WERAnalyzer.cs

```diff
@@ -79,17 +79,20 @@ private void AllExternalTools_CollectionChanged(object? sender, NotifyCollection
         // with it.
         if (e.Action == NotifyCollectionChangedAction.Add && e.NewItems is not null)
         {
+            List<Tool> newAnalysisTools = new();
```

Comment:
Curious - would it be useful to keep this list as a field in the class rather than creating/destroying it on every change?

Reply:
This list only contains the newly added tools that we'd want to run on all of our existing cabs. There is a list of all the analysis tools (stored in _registeredAnalysisTools) that run when new cabs come in. We just don't want to run all of the analysis tools when a new tool is added.